jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/AbstractNetworkConnector.java

```diff
@@ -85,11 +85,55 @@ protected void doStop() throws Exception
     @Override
     public void open() throws IOException
     {
+        notifyOpen();
+    }
+
+    private void notifyOpen()
+    {
+        getEventListeners().stream()
+            .filter(l -> l instanceof NetworkConnector.Listener)
+            .map(NetworkConnector.Listener.class::cast)
+            .forEach(this::notifyOpen);
+    }
+
+    private void notifyOpen(NetworkConnector.Listener listener)
+    {
+        try
+        {
+            listener.onOpen(this);
+        }
+        catch (Throwable x)
+        {
+            if (LOG.isDebugEnabled())
+                LOG.debug("failure while notifying listener {}", listener, x);
+        }
```

Comment:
I'm not sure streams and method handlers are helping code clarity here.  I think a good old for loop would be better than 2 other methods

```suggestion
        for (EventListener l : getEventListeners())
        {
            if (l instanceof NetworkCommector.Listener ncl)
            {
                try
                {
                    ncl.onOpen(this);
                }
                catch (Throwable x)
                {
                    if (LOG.isDebugEnabled())
                         LOG.debug("failure while notifying listener {}", ncl, x);
                }
            }
```